Resources/Textures/DeltaV/Objects/Consumable/Drinks/drgibbbloodred.rsi/meta.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": 1,
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "https://github.com/discordia-space/CEV-Eris/raw/9c980cb9bc84d07b1c210c5447798af525185f80/icons/obj/food.dmi, Edit by Yuukitten",
+    "states": [
+      {
+        "name": "icon"
+      },
+      {
+        "name": "icon_open"
+      },
+      {
+        "name": "inhand-right",
+        "directions": 4,
+        "delays": [
+          [
+            1
+          ],
+          [
+            1
+          ],
+          [
+            1
+          ],
+          [
+            1
+          ]
+        ]
```

Comment:
why does this need to exist

Reply:
Rotation, its on literally all other held item cans

Reply:
rotation is directions not delays, this isn't needed at all